src/contract-details/ContractChart.js

```diff
@@ -67,35 +66,43 @@ export default class ContractChart extends PureComponent {
     })[timePeriod];
     const type = timePeriod === '1t' ? 'line' : 'candlestick';
 
-    this.chart = wtcharts.chartWindow.addNewChart($(this.root), {
-      type, // default is 'line'
-      timePeriod, // default is '1t'
-      instrumentCode: underlying,
-      instrumentName: symbolName,
-      showInstrumentName: true, // default is false
-      showOverlays: false, // default is true
-      showShare: false, // default is true
-      start: (purchase_time || date_start) - margin,
-      end: sell_time && +sell_time + margin || (exit_tick_time ? exit_tick_time + margin : 'latest'),
-      hideCurrentPrice: true,
-      // timezoneOffset: 0,
-    });
+    (async () => {
```

Comment:
@brucebinary use async lifecycle methods instead of an iife, see [this blog post](https://medium.com/front-end-hacking/async-await-with-react-lifecycle-methods-802e7760d802)